src/maintainer/maintainer_faq.rst

```diff
@@ -67,7 +67,9 @@ FAQ
 
 :ref:`(Q) <mfaq_anaconda_delay>` **Why does my new version appear on Anaconda Cloud, but is not installable with conda?**
 
-  For certain, high-traffic channels (main & conda-forge), Anaconda uses a `CDN <https://cloudflare.com/learning/cdn/what-is-a-cdn/>`__ to decrease costs. The CDN is only reindexed every 20 minutes, however `Anaconda.org <https://anaconda.org>`__ uses the original channel that the CDN mirrors.  Therefore, packages will show up on the `Anaconda Cloud <https://anaconda.org>`__ about 20 to 40 minutes before they are downloadable via conda.  You can use ``conda search <pkg>``  to see if the package is installable, because this command reads from the CDN.
+  For certain, high-traffic channels (main & conda-forge), Anaconda uses a `CDN <https://cloudflare.com/learning/cdn/what-is-a-cdn/>`__ to decrease costs. Therefore, packages will show up on the `Anaconda Cloud <https://anaconda.org>`__ about 10 minutes before they are downloadable via conda.  You can use ``conda search <pkg>``  to see if the package is installable, because this command reads from the CDN.
+
+  If the CDN sync is not happening timely, please check the `status page <https://conda-forge.org/status/>`_ for the CDN cloning status and see if any problem occurs. If the clone is not sync'ing, you can open a CDN Issue in the `Anaconda Issues repo <https://github.com/ContinuumIO/anaconda-issues>`_.
```

Comment:
Nice! We should also mention this repo: https://github.com/conda/infra/discussions

Reply:
Interesting, I didn't know this, so which repo should one report to?